### PR TITLE
Created evaluation_metrics folder for classification

### PR DESCRIPTION
**Pull request by NEERAJAP2001**
1. This folder is necessary for adding more algorithms in classification as we need to import metrics there.

2. This PR is basically for solving a part of #21 as we have algorithms coming there up.

3. I have added the tests also.

4. This pull request is a part of NJACK Winter of Code so please merge this after 1 Dec 2020 so that it gets counted.

5. Any suggestions for changes are always welcome.

### CODE REVIEW

File: datascientist/model/classification/evaluation_metrics/accuracy.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    y : numpy array
+    Actual value.
+    y_hat : numpy array
+    Predicted value.
+    """
+    
+    final_acc=accuracy_score(y, y_hat)
+    return final_acc
+
+
```
Comment (sidhu1012) [inline datascientist/model/classification/evaluation_metrics/accuracy.py:18]: I don't think classification needs evaluation metrics files since accuracy(y, y_hat) is better than call the private function.

File: datascientist/model/classification/skl/logistic_regression.py [hunk 2]
```diff
@@ -0,0 +1,30 @@
+from datascientist.model.classification.evaluation_metrics.accuracy import _accuracy
+
+from sklearn.linear_model import LogisticRegression
+import numpy as np
+
+
+
+def _logistic_regression(*, train, test,x_predict=None,metrics=None, n_jobs=None):
```
Comment (sidhu1012) [inline datascientist/model/classification/skl/logistic_regression.py:8]: These parameters are incomplete, please check how Regression functions are made.

Comment (NEERAJAP2001) [reply datascientist/model/classification/skl/logistic_regression.py:8]: yeah!! I'm still editing that !!

File: datascientist/model/classification/evaluation_metrics/accuracy.py (hunk 1 above)
Comment (NEERAJAP2001) [reply datascientist/model/classification/evaluation_metrics/accuracy.py:18]: So, what should I do to find the accuracy of (y,y_hat) sir?

Comment (sidhu1012) [reply datascientist/model/classification/evaluation_metrics/accuracy.py:18]: Directly acc = accuracy_score(y, y_hat).

Don't call me sir , or anyone sir in open source , adress with username.

Add atleast 3 accuracy methods possibly jaccard, f1 and accuracy.

**Review by sidhu1012 — COMMENTED** [review]
R

File: datascientist/model/classification/evaluation_metrics/accuracy.py (hunk 1 above)
Comment (NEERAJAP2001) [reply datascientist/model/classification/evaluation_metrics/accuracy.py:18]: okay @sidhu1012  so, shall I make files for  Jaccard, f1, or  simply call them like this? 


> Directly acc = accuracy_score(y, y_hat).

Comment (sidhu1012) [reply datascientist/model/classification/evaluation_metrics/accuracy.py:18]: Simply call them

File: datascientist/model/classification/skl/logistic_regression.py (hunk 2 above)
Comment (NEERAJAP2001) [reply datascientist/model/classification/skl/logistic_regression.py]: @sidhu1012  can you tell me what are the missing parameters? I'll add them up !!

**Comment by sidhu1012** [comment]
@NEERAJAP2001 ping me when it's finished!

**Comment by NEERAJAP2001** [comment]
yeah @sidhu1012 I'll be adding the algorithm soon!! can you help me here 
> @sidhu1012 can you tell me what are the missing parameters? I'll add them up !!
https://github.com/Dacience/DataScientist/pull/46#discussion_r533037116

**Comment by sidhu1012** [comment]
> yeah @sidhu1012 I'll be adding the algorithm soon!! can you help me here
> 
> > @sidhu1012 can you tell me what are the missing parameters? I'll add them up !!
> > [#46 (comment)](https://github.com/Dacience/DataScientist/pull/46#discussion_r533037116)

https://github.com/Dacience/DataScientist/blob/bb8c863a01640f8e8baae9b57fe9c79267f08ed2/datascientist/model/regression/skl/linear_model/linear_regression.py#L9

Till `metrics` parameters must be same , then the parameters of model should follow.

**Comment by NEERAJAP2001** [comment]
oh! okay cool 
thanks!

**Comment by NEERAJAP2001** [comment]
Okay !! I have pulled here #48

**Comment by sidhu1012** [comment]
Continued in #48
